data/json/monster_factions.json

```diff
@@ -123,5 +123,20 @@
         "type" : "MONSTER_FACTION",
         "name" : "spider_cellar",
         "base_faction" : "spider"
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant",
+        "by_mood" : ["small_animal", "blob", "utility_bot", "mutant_omnivore", "mutant_predator"]
```

Comment:
You don't need to specify "by_mood" to things that are already set to that. In this case, everything is already hostile towards small animals, blobs and utility bots, because "by_mood" is the default attitude towards everything.